dynamic_programming/bitmask.py

```diff
@@ -11,75 +11,54 @@
 from collections import defaultdict
 
 
-class AssignmentUsingBitmask:
-    def __init__(self, task_performed, total):
-        self.total_tasks = total  # total no of tasks (N)
-
-        # DP table will have a dimension of (2^M)*N
-        # initially all values are set to -1
+class TaskAssignment:
+    def __init__(self, task_performed, total_tasks):
```

Comment:
No type hint, no doctests?

Reply:
This is not resolved.  Please read CONTRIBUTING.md.  Functions need type hints and doctests.